pkg/pkg/common/common.go

```diff
@@ -38,3 +43,69 @@ func PrintProcessedPackages(update []Package, action string) {
 		fmt.Printf("  - %s\n", pkg.String())
 	}
 }
+
+// GenerateRelativePath creates a path to navigate back to the root directory
+// from the specified outputFolder
+func GenerateRelativePath(outputFolder string) string {
+	outputFolder = strings.TrimRight(outputFolder, "/")
+	dirCount := strings.Count(outputFolder, "/") + 1
+	path := ""
+	for i := 0; i < dirCount; i++ {
+		path += "../"
+	}
+	return strings.TrimRight(path, "/")
+}
+
+func fileExists(path string) (bool, error) {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	return !info.IsDir(), nil
+}
+
+func dirExists(path string) (bool, error) {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	return info.IsDir(), nil
+}
+
+func UseConsolidatedPackageStructure(dir string) (bool, error) {
+	packagePath := filepath.Join(dir, PackagesManifestFilename)
+	configDir := filepath.Join(dir, BoilerplatePackageTerraformConfigPrefix)
```

Comment:
```suggestion
	varFileDir := filepath.Join(dir, BoilerplatePackageTerraformConfigPrefix)
```

Jeg prøver å unngå å bruke navnet "config" i ok-koden, fordi det kan bety så mange ting. Har sett at koden for skjemahåndtering har blitt vanskelig å forstå pga det. I stedet tror jeg domenenavnet bør være "var file", derav "varFileDir".